src/Sylius/Component/User/Model/AdvancedUserInterface.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Component\User\Model;
+
+use Symfony\Component\Security\Core\User\UserInterface as SymfonyUserInterface;
+
+interface AdvancedUserInterface extends SymfonyUserInterface
```

Comment:
This interface needs `@see` with reference to original interface and explanation why deprecated interface is added back to sylius.